### ISSUE TITLE
[TW-1824] countdown (date format) is not working properly

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1840]
_Eric Hymowitz on 2016-07-04T01:21:34Z says:_ 

 The "countdown" date format is described in the documentation (and from Paul's email) as being an ISO-8601-formatted time duration for the length of time until ... whatever (for example, "due.countdown" is how much time until the task is due.)

In reality, the "due.countdown" appears to be incorrectly defined as "the negative of due.relative but never less than zero" or "the amount of time elapsed since the due date" and formatted the same way as "due.relative".

(I have not tested other date fields with countdown)

Replicate problem thusly:

1. Have some tasks that are both completed/overdue and waiting/not yet completed.

2. Create a new report with these lines in .taskrc:
report.cccc.columns=id,due.relative,due,due.countdown,description
report.cccc.labels=ID,Rel,DueDate,Cdn,Description

3. Run the report

```
herman-pts/0:~% task cccc

ID Rel  DueDate    Cdn Description               
 1 -6w  2016-05-19  6w check the patches         
 3 -5w  2016-05-24  5w install the backup disk   
 4 -4w  2016-05-31  4w last day of the month     
 9 1d   2016-07-05     install the backup disk   
10 3d   2016-07-07     check the patches         
11 3w   2016-07-30     last day of the month
```

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-07-04T01:21:34Z
Modified: 2016-07-04T16:12:45Z
```

**Comment by taskwarrior** [comment]
_Daniel Shahaf on 2016-07-04T01:46:35Z says:_ 

 (I've taken the liberty of adding ` noformat ` tags to make the table easier to read.)

**Comment by taskwarrior** [comment]
_Eric Hymowitz on 2016-07-04T16:12:45Z says:_ 

 Thank you.  I couldn't find an option to do that for multiple lines, just one line at a time.

**Comment by sebu06** [comment]
Hi,
I think the error is in ColumnTypeDate::render() (ColTypeDate.cpp:161):
```
    else if (_style == "countdown")
    {
      Datetime now;
      renderStringRight (lines, width, color, Duration (now - date).formatVague (true));
    }
 ```
 however, I think it should be `date - now` instead. However, I'm not completely sure if I dont misunderstand the intention of the countdown column here. Otherwise I could create the pull request for this.

**Comment by tbabej** [comment]
Hi @sebu06,

I think this is a correct assessment of the problem. Countdown, as the name suggests, should show the duration remaining to complete the task.

Feel free to go for a PR! :slightly_smiling_face:

**Comment by sebu06** [comment]
fixed with PR #2542
